db/seed.js

```diff
@@ -0,0 +1,43 @@
+const faker = require('faker');
+const db = require('./index.js');
+const { connection } = require('./index.js').connection;
+
+const seedUsers = async () => {
+  for (let i = 0; i < 100; i++) {
+    const user = { username: faker.internet.userName() };
+    await db.insertUser(user);
+  }
+};
+
+// const seedUsers = async () => {
+//   let promisesToAwait = [];
+//   for (let i = 0; i < 100; i++) {
+//     let user = { username: faker.internet.userName() };
+//     promisesToAwait.push(db.insertUser(user));
+//     // await db.insertUser(user);
+//   }
+//   await Promise.all(promisesToAwait);
+// };
+
+
+const seedReviews = async () => {
+  for (let i = 0; i < 100; i++) {
+    const review = {};
+    review.date = faker.date.past();
+    review.review = faker.lorem.paragraph();
+    review.rating = (Math.floor(Math.random() * 5) + 1);
+    review.book_id = (Math.floor(Math.random() * 100) + 1);
```

Comment:
I agree that you might not need a book table since you are not using any details from the books for the reviews table than a randomly assigned id number. However, I think the fec goal is to generate mock data for 100 items (or books in our case). In that case, I think you may want to generate more than 100 reviews to more accurately mock the data for 100 books.